builtin/providers/aws/resource_aws_rds_cluster_instance.go

```diff
@@ -83,6 +83,20 @@ func resourceAwsRDSClusterInstance() *schema.Resource {
 				Computed: true,
 			},
 
+			"kms_key_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
```

Comment:
This should probably just be `computed`, right? Because we can't set this here, correct?